net/etesync-server/files/uci.cfg

```diff
@@ -0,0 +1,13 @@
+
+config direct 'global'
```

Comment:
Can `direct` and `indirect` be changed to more meaningful names, or removed entirely? I know they had a function previously but now I don't believe they have a purpose.

Reply:
Thank you, I will change it :-)